curation_concerns-models/app/jobs/characterize_job.rb

```diff
@@ -1,9 +1,12 @@
 class CharacterizeJob < ActiveFedoraIdBasedJob
   queue_as :characterize
 
+  # Calls characterization, saves the generic_file, and queues a job to create derivatives.
+  # Sets original_name property to be same as that of original_file thus over riding characterization.
   def perform(id)
     @id = id
-    CurationConcerns::CharacterizationService.run(generic_file)
```

Comment:
This change is at cross purposes with #347 What's the best way for us to resolve that?